04_datainfra.Rmd

```diff
@@ -409,6 +409,167 @@ molten_se <- meltAssay(se,
 molten_se
 ```
 
+### Subsetting
+
+**Subsetting** data helps to draw the focus of analysis on particular sets of samples and / or features. When dealing with large data sets, the subset of interest can be extracted and investigated separately. This might improve performance and reduce the computational load.
+
+Load:
+
+* mia
+* dplyr
+* knitr
+* data `GlobalPatterns`
+
+```{r include = FALSE}
+# load libraries and data
+library(mia)
+library(dplyr)
+library(knitr)
+```
+
+Let us store `GlobalPatterns` into `se` and check its original number of features (rows) and samples (columns). **Note**: when subsetting by sample, expect the number of columns to decrease; when subsetting by feature, expect the number of rows to decrease.
+
+```{r}
+# store data into se and check dimensions
+data("GlobalPatterns")
+se <- GlobalPatterns
+# show dimensions
+dim(se) ## [1] n°features    n°samples
+```
+
+#### Subset by sample (column-wise)
+
+For the sake of demonstration, here we will extract a subset containing only the samples of human origin (feces, skin or tongue), stored as `SampleType` within `colData(se)`.
+
+First, we would like to see all the possible values that `SampleType` can take on and how frequent those are: 
+
+```{r}
+# inspect possible values for SampleType
+unique(colData(se)$SampleType)
```

Comment:
I think `unique(se$SampleType)` gives the same and is shorter. Check & change here and elsewhere